test/components/BaseForm_test.js

```diff
@@ -127,4 +127,48 @@ describe("BaseForm component", () => {
     expect(submit.disabled).toBe(true);
     expect(result.queryByTestId("spinner")).toBeDefined();
   });
+
+  it("Should scroll to the first property that fails validation", async () => {
+    const result = render(
+      <BaseForm
+        className="testClass"
+        schema={testSchema}
+        uiSchema={testUiSchema}
+        onSubmit={jest.fn()}
+        customValidate={(data, errors) => {
+          errors.title.addError("test error");
+          return errors;
+        }}
+      />
+    );
```

Comment:
Shall we add a test here to make sure the query selector that we use in the code is found in the page? `.querySelector(`[for="root${err.property.replace(/\./g, "_")}"]`)`
This way if RJSF changes the way it names its fields, the test would break

Reply:
Good call, added